tools/test-proxy/Azure.Sdk.Tools.TestProxy/Common/RecordMatcher.cs

```diff
@@ -34,49 +34,36 @@ public RecordMatcher(bool compareBodies = true, bool ignoreQueryOrdering = false
             _ignoreQueryOrdering = ignoreQueryOrdering;
         }
 
+        /// <summary>
+        /// Headers that will be entirely ignored during matching. 
+        /// </summary>
         public HashSet<string> ExcludeHeaders = new HashSet<string>(StringComparer.OrdinalIgnoreCase)
+        {
+            "Request-Id",
+            "traceparent",
+        };
+
+
+        /// <summary>
+        /// Headers whose CONTENT will be ignored during matching, but whose PRESENCE will still be checked for on both request and record sides. 
+        /// </summary>
+        public HashSet<string> IgnoredHeaders = new HashSet<string>(StringComparer.OrdinalIgnoreCase)
         {
             "Date",
             "x-ms-date",
             "x-ms-client-request-id",
             "User-Agent",
             "x-ms-useragent",
             "x-ms-version",
-            "Request-Id",
-            "traceparent",
             "If-None-Match",
             "sec-cha-ua",
             "sec-ch-ua-mobile",
             "sec-ch-ua-platform",
             "Referrer",
+            "Referer",
```

Comment:
Here.

Reply:
Adding this because of the fact that this is [actually a thing.](https://en.wikipedia.org/wiki/HTTP_referer)